src/vs/workbench/contrib/terminal/browser/xterm/xtermTerminal.ts

```diff
@@ -234,6 +237,10 @@ export class XtermTerminal extends DisposableStore implements IXtermTerminal, II
 		} else if (this._shouldLoadCanvas()) {
 			this._enableCanvasRenderer();
 		}
+		// Enable addons
+		if (this._configHelper.config.experimental.fontLigatures) {
```

Comment:
aren't there problems w `configHelper` such that `configurationService` should be used instead?

Reply:
Config helper itself is fine, it's just a debt item to remove in favor of configuration service as it doesn't add much now